tests/vmi_configuration_test.go

```diff
@@ -1150,31 +1148,21 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 				By("Checking that the VM memory equals to a number of consumed hugepages")
 				Eventually(func() bool { return verifyHugepagesConsumption() }, 30*time.Second, 5*time.Second).Should(BeTrue())
 			},
-				Entry("[Serial][test_id:1671]hugepages-2Mi", Serial, "2Mi", "64Mi", "None"),
-				Entry("[Serial][test_id:1672]hugepages-1Gi", Serial, "1Gi", "1Gi", "None"),
-				Entry("[Serial][test_id:1672]hugepages-2Mi with guest memory set explicitly", Serial, "2Mi", "70Mi", "64Mi"),
+				Entry("[Serial][test_id:1671]hugepages-2Mi", decorators.RequiresHugepages2Mi, Serial, "2Mi", "64Mi", "None"),
+				Entry("[Serial][test_id:1672]hugepages-1Gi", decorators.RequiresHugepages1Gi, Serial, "1Gi", "1Gi", "None"),
+				Entry("[Serial][test_id:1672]hugepages-2Mi with guest memory set explicitly", decorators.RequiresHugepages2Mi, Serial, "2Mi", "70Mi", "64Mi"),
 			)
 
 			Context("with unsupported page size", func() {
 				It("[test_id:1673]should failed to schedule the pod", func() {
```

Comment:
Doesn't this require the decorator?

Reply:
This actually don't need any special setup.